.gitignore

```diff
@@ -37,6 +37,7 @@ restart_timestamp
 
 # Text files (For statistical output from ocean model)
 *.txt
+!CMakeLists.txt
```

Comment:
what is this?

Reply:
This prevents the .gitignore from ignoring the `CMakeLists.txt` file which matches with the generic *.txt ignore that was already there.